app/controllers/IneligibleController.scala

```diff
@@ -185,10 +187,12 @@ class IneligibleController @Inject() (
     )
 
   val epayeYouHaveChosenNotToSetUpPage: Action[AnyContent] = as.authenticatedJourneyAction { implicit request =>
+    auditService.auditDdInProgress(request.journey, false)
```

Comment:
it'd be nice to add in something in the unit tests which checks that the correct thing is audited whenever you trigger an event like this. Here, in the spec which checks the content of the page you could add in something there for instance. There should be something in the test `AuditStub` already which already allows you to check that an audit event looks correct.